main/index.js

```diff
@@ -33,6 +37,21 @@ const initializePlugins = async () => {
   }
 };
 
+const checkForUpdates = () => {
+  if (is.development) {
+    return false;
+  }
+
+  autoUpdater.logger = log;
+  autoUpdater.logger.transports.file.level = 'info';
```

Comment:
This is so we can debug auto-update functionality in Console.app

Reply:
Maybe add a comment for the future?